README.md

```diff
@@ -56,6 +56,22 @@ Key : `mediaSection.location`
 Example : `file:/home/user/customMediaSection.json`  
 Default : `classpath:public/json/media.json`
 
+### Override Users' Profile (optional)
+
+You can override the users' profile informations by setting the `profiles.location` property to your config file.
```

Comment:
Should be `user's` instead of `users'`

Reply:
It is users in plurial, so it is correct